.travis.yml

```diff
@@ -6,6 +6,7 @@ rvm:
   - '2.3'
```

Comment:
Don't know if we still want to test against 2.3? Both it and 2.4 have been EoL for a while after all.

Reply:
This would be a different PR.